command/v7/app_command_test.go

```diff
@@ -203,7 +203,7 @@ var _ = Describe("app Command", func() {
 						},
 					},
 					CurrentDroplet: resources.Droplet{
-						Stack: "cflinuxfs2",
```

Comment:
question: I've noticed on multiple occasions both `cflinux2` and `cflinux3` were replaced with `cflinux4`. Do we know why we had two different stacks before?

Reply:
Some tests had them. Different branches had different values